docs/checklist.md

```diff
@@ -27,6 +27,48 @@ Comes in three sections. You need varying degrees of access depending on where y
+
+**NOTE**: `Contributor`s are only able to edit _non-published_ stories. Once a story is published, many cards will be hidden from `contributor`s as well.
+
+This is outlined in the table below:
+
+|Story issue|Users that will be shown the checklist card|Users that will be shown the checklist card when the story has been published|
+|--|--|--|
+|Story missing title|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Story title too long|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Story missing excerpt|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Story has no poster image attached|Super Admin, Administrator, Editor, Author|Super Admin, Administrator, Editor, Author|
+|Increase poster image's size|Super Admin, Administrator, Editor, Author|Super Admin, Administrator, Editor, Author|
+|Increase poster images's aspect ratio|Super Admin, Administrator, Editor, Author|Super Admin, Administrator, Editor, Author|
+|Increase publisher logo size|Super Admin, Administrator, Editor, Author|Super Admin, Administrator, Editor, Author|
+|Video missing poster image|Super Admin, Administrator, Editor, Author|Super Admin, Administrator, Editor, Author|
+|Video not optimized|Super Admin, Administrator, Editor, Author|Super Admin, Administrator, Editor, Author|
+|Story has too many pages|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Story has too few pages|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Page has too much text|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Page has too little text|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Page has too many links|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Video element resolution too low|Super Admin, Administrator, Editor, Author|Super Admin, Administrator, Editor, Author|
+|Media element resolution too low|Super Admin, Administrator, Editor, Author|Super Admin, Administrator, Editor, Author|
+|Contrast too low between text and background color|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Text font size too small|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Video element missing description|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
+|Video element missing captions|Super Admin, Administrator, Editor, Author, Contributor|Super Admin, Administrator, Editor, Author|
```

Comment:
Contributors typically can't upload media like a captions file

Reply:
Updated this to use permissions instead of roles - should be more accurate now